requirements/prod.txt

```diff
@@ -4,4 +4,5 @@ types-PyYAML>=6.0.11
 types-requests>=2.28.9
 python-dateutil>=2.8.2
 requests>=2.28.1
-ndjson>=0.3.1
\ No newline at end of file
+ndjson>=0.3.1
```

Comment:
which reminds me, shall we have a PR to just update all the dependencies?

Reply:
This was already made and merged #71.

I will rebase this on there.

Reply:
Done.